packages/aws-cdk/lib/init-templates/sample-app/python/app.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+from aws_cdk import cdk
+
+from hello.hello_stack import MyStack
+
+
+app = cdk.App()
+MyStack(app, "hello-cdk-1")
```

Comment:
Maybe put each stack in a different region (would be a nicer example)

Reply:
Done.